examples/with-linaria/package.json

```diff
@@ -7,9 +7,10 @@
     "start": "next start"
   },
   "dependencies": {
-    "@zeit/next-css": "^1.0.1",
-    "linaria": "2.0.0-alpha.5",
+    "@babel/core": "^7.13.10",
```

Comment:
Is babel needed here?

Reply:
Yep, it's a peer dependency of `linaria`. Without it the build errors,
```
error - ./pages/index.js
Error: Cannot find module '@babel/core'
Require stack:
- /Users/turadg/Code/next.js/examples/with-linaria/node_modules/linaria/lib/transform.js
- /Users/turadg/Code/next.js/examples/with-linaria/node_modules/linaria/lib/loader.js
```